test/agentchat/test_function_call_groupchat.py

```diff
@@ -59,5 +59,62 @@ def get_random_number():
     user_proxy.initiate_chat(manager, message="Let's start the game!")
 
 
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.10"),
+    reason="do not run if openai is not installed or py!=3.10",
+)
+def test_update_function():
```

Comment:
This test is not about group chat, right? Can you put this test to the other function call test file? i.e., `test_function_call.py`

Reply:
resolved